sway/tree/output.c

```diff
@@ -102,20 +102,19 @@ struct sway_output *output_create(struct wlr_output *wlr_output) {
 	output->workspaces = create_list();
 	output->current.workspaces = create_list();
 
+	size_t len = sizeof(output->layers) / sizeof(output->layers[0]);
+	for (size_t i = 0; i < len; ++i) {
+		wl_list_init(&output->layers[i]);
+	}
+
 	return output;
 }
 
 void output_enable(struct sway_output *output) {
```

Comment:
Hmm, maybe rename this to `configure_output`?

Reply:
Renamed `output_enable` to `output_configure`